jQuery v3 types are looking for Sizzle types

Found when testing types against Kitchensink https://circleci.com/gh/cypress-io/cypress/158000

```
> cypress-example-kitchensink@0.0.0-development types /tmp/kitchensink
> tsc --noEmit

node_modules/cypress/types/jquery/index.d.ts:28:23 - error TS2688: Cannot find type definition file for 'sizzle'.

28 /// <reference types="sizzle" />
                         ~~~~~~

node_modules/cypress/types/jquery/misc.d.ts:27:33 - error TS2503: Cannot find namespace 'Sizzle'.

27     interface Selectors extends Sizzle.Selectors {
                                   ~~~~~~

node_modules/cypress/types/jquery/misc.d.ts:35:14 - error TS2503: Cannot find namespace 'Sizzle'.

35         ':': Sizzle.Selectors.PseudoFunctions;
                ~~~~~~

node_modules/cypress/types/jquery/misc.d.ts:43:17 - error TS2503: Cannot find namespace 'Sizzle'.

43         filter: Sizzle.Selectors.FilterFunctions;
                   ~~~~~~


Found 4 errors.
```

--- a/cli/package.json
+++ b/cli/package.json
@@ -28,6 +28,7 @@
   "dependencies": {
     "@cypress/listr-verbose-renderer": "0.4.1",
     "@cypress/xvfb": "1.2.4",
+    "@types/sizzle": "2.3.2",
     "arch": "2.1.1",
     "bluebird": "3.5.0",
     "cachedir": "1.3.0",

--- a/cli/types/index.d.ts
+++ b/cli/types/index.d.ts
@@ -19,6 +19,10 @@
 /// <reference path="./jquery/index.d.ts" />
 /// <reference path="./chai-jquery/index.d.ts" />
 
+// jQuery includes dependency "sizzle" that provides types
+// so we include it too in "node_modules/sizzle".
+// This way jQuery can load it using 'reference types="sizzle"' directive
+
 // "moment" types are with "node_modules/moment"
 /// <reference types="moment" />
 

